src/test/rustdoc/no-compiler-reexport.rs

```diff
@@ -1,7 +1,9 @@
-// compile-flags: --no-defaults
+// compile-flags: -Z unstable-options --document-hidden-items --document-private-items
 
 #![crate_name = "foo"]
 
-// @has 'foo/index.html' '//code' 'extern crate std;'
-// @!has 'foo/index.html' '//code' 'use std::prelude::v1::*;'
+// no way to get this output without --no-defaults / #![doc(no_default_passes)]
+// which are now non-functional
```

Comment:
I would just remove this comment here, I don't think it's necessary. Thanks for looking into it though!